### PR TITLE
deploy: Fix witnesses for Balance and Container contract deployments

### PR DESCRIPTION
**Pull request by cthulhu-rider**
tested with 1/4/7 node envs

### CODE REVIEW

File: pkg/morph/deploy/contracts.go [hunk 1]
```diff
@@ -126,11 +135,54 @@ func syncNeoFSContract(ctx context.Context, prm syncNeoFSContractPrm) (util.Uint
...
-		deployCommitteeActor, err := newCommitteeNotaryActorWithCustomCommitteeSigner(prm.blockchain, prm.localAcc, prm.committee, func(s *transaction.Signer) {
-			s.Scopes = transaction.CustomContracts
-			s.AllowedContracts = append(prm.extraCommitteeDeployAllowedContracts, prm.nnsContract)
-		})
+	if prm.deployWitness > 0 {
+		if prm.deployWitness != witnessAlphabet && prm.deployWitness != witnessAlphabetAndCommittee {
+			panic(fmt.Sprintf("unexpected deploy witness mode value %v", prm.deployWitness))
+		}
+
+		committeeDefaultMultiSigAcc := wallet.NewAccountFromPrivateKey(prm.localAcc.PrivateKey())
```
Comment (roman-khimov) [inline pkg/morph/deploy/contracts.go:143]: Technically, it's not a committee address. It's a validator multisig (when you take validators only, of course, so `ValidatorsCount` can be applied, although NeoFS has it set to the size of the committee). Your initial NEO and GAS reside on this account, for example.

File: pkg/morph/deploy/contracts.go [hunk 2]
```diff
@@ -126,11 +135,54 @@ func syncNeoFSContract(ctx context.Context, prm syncNeoFSContractPrm) (util.Uint
...
+		signers[0].Signer.Account = prm.localAcc.ScriptHash()
+		signers[0].Signer.Scopes = transaction.None
+		// Alphabet
+		signers[1].Account = committeeDefaultMultiSigAcc
+		signers[1].Signer.Account = committeeDefaultMultiSigAcc.ScriptHash()
+		signers[1].Signer.Scopes = transaction.CustomContracts
+		signers[1].Signer.AllowedContracts = prm.alphabetDeployAllowedContracts
+
+		if prm.deployWitness == witnessAlphabetAndCommittee {
+			committeeMajorityMultiSigAcc := wallet.NewAccountFromPrivateKey(prm.localAcc.PrivateKey())
```
Comment (roman-khimov) [inline pkg/morph/deploy/contracts.go:161]: And this is a proper committee, as is, not some special "majority" one.

**Review by roman-khimov — COMMENTED** [review]
The fix is OK, but we better improve the terminology at the same time, otherwise we'll leave some misconceptions in the code.

File: pkg/morph/deploy/contracts.go (hunk 1 above)
Comment (cthulhu-rider) [reply pkg/morph/deploy/contracts.go:143]: ok i agree, competely do not like inconsistency of terms

btw what u think of Alphabet? it's mentioned in many places like https://pkg.go.dev/github.com/nspcc-dev/neofs-contract@v0.19.1/netmap#SubscribeForNewEpoch

Comment (roman-khimov) [reply pkg/morph/deploy/contracts.go:143]: I doubt NeoFS will survive in `len(committee) != ValidatorsCount` scenario now, btw. But it's a bit different story.

Comment (cthulhu-rider) [reply pkg/morph/deploy/contracts.go]: left "validators" for now as more Neo-known term

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**